osgi-dependencies/proxy/src/main/java/com/cognifide/aet/proxy/ProxyServerProvider.java

```diff
@@ -44,19 +44,20 @@
           unbind = "unbindProxyManager")
   private final Map<String, ProxyManager> collectorManagers = Maps.newConcurrentMap();
 
-  public ProxyServerWrapper createProxy(String useProxy, int port) throws ProxyException {
-    String proxyType = useProxy;
-    if ("true".equals(useProxy)) {
+  public ProxyServerWrapper createProxy(String usedProxy) throws ProxyException {
+    String proxyType = usedProxy;
+    if ("true".equals(usedProxy)) {
```

Comment:
Do I understand correctly you don't like having `"true"` passed as sting type value?
I have renamed the variables in order to make the code more clear.
However:
- it may be bad idea to use **enum** for proxy types, as we don't know how many type there will be in an AET system. someone could add new `ProxyManager` service with new type.
- the `"true"` value is passed from suite XML configuration but the [`useProxy` attribute](https://github.com/Cognifide/aet/wiki/SuiteStructure#proxy) could have non-boolean values.

I can't make this code better :( Could you provide some code samples / commits / pull requests?